### DISCUSSION TITLE
How to create a heatmap?

### DISCUSSION THREAD

**Discussion by Roberto-Pecora** [atoti/atoti#482] [Q&A] [answered]
Hi all, 
how can I create a heatmap in Atoti?

Thanks

**Comment by bloussou** [comment, marked as answer]
Hello,

We don't have heatmaps (yet). You could write your own by developing an [`app_extension`](https://docs.atoti.io/0.6.4/lib/atoti.config.session_config.html#atoti.config.session_config.SessionConfig.app_extensions).
